AuroraEditor/Features/Version Control/Model/Rules/Branch/RepoBranchRules.swift

```diff
@@ -60,6 +60,7 @@ struct RepoBranchRules {
 
                 // there are no relevant rules for this branch name, so return
                 if toCheck.isEmpty {
+                    print("No relevant rules for this branch name")
```

Comment:
If not temporarily then please make this an NSLog / Logger.

Reply:
Maybe it's better to remove the exception for the version control folder to have same code style anywhere, this could have been caught by swiftlint.